<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connection_window</class>
 <widget class="QMainWindow" name="connection_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Connection Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="connection_name_txt"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Access Key Id</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="access_id_txt">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Access Secret Key</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="secret_key_txt">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Region</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="region_cmb"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="dialog_box">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>299</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_box</sender>
   <signal>rejected()</signal>
   <receiver>connection_window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_box</sender>
   <signal>accepted()</signal>
   <receiver>connection_window</receiver>
   <slot>save_connection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
